src/index.html

```diff
@@ -1,14 +1,23 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Snake</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Snake</h1>
-</body>
+  <head>
```

Comment:
![image](https://user-images.githubusercontent.com/38065883/172372178-cb40699f-f9f7-48cd-8b72-08f836ab5258.png)
Blocks shouldn't be less then 300px wide

Reply:
https://user-images.githubusercontent.com/38065883/172704077-3d2debb3-8c02-44b0-8975-874dd93eec69.mov

It gets broken on some screens